ocs_ci/ocs/cluster.py

```diff
@@ -3316,3 +3316,68 @@ def check_cephcluster_status(
             f' {cc_resource["status"]["ceph"]["health"]}'
         )
         raise CephHealthException()
+
+
+def get_active_mds_info():
+    """Return information about the active Ceph MDS.
+
+    Returns:
+        dict: A dictionary containing information about the active MDS daemon,
+        including the following keys in case of success, otherwise None.
+        - "node_ip": The IP address of the node running the active MDS daemon.
+        - "mds_daemon": The name of the MDS daemon.
+        - "active_pod": The name of the active pod.
+        - "node_name": The name of the node where active mds pod is running.
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    ceph_mdsmap = ct_pod.exec_ceph_cmd("ceph fs status")
+
+    # Find ceph daemon state as 'active'
+    ceph_daemon_name = next(
+        (
+            daemon["name"]
+            for daemon in ceph_mdsmap["mdsmap"]
+            if daemon["state"] == "active"
+        ),
+        None,
+    )
+
+    if ceph_daemon_name is None:
+        logger.error("No active MDS daemon found")
+        return None
+
+    logger.info(f"Found active MDS daemon: {ceph_daemon_name}")
+
+    # Find ceph MDS pod running 'active' daemon.
```

Comment:
all these comments can be removed, instead convert it to log message

Reply:
done